src/layouts/Sites.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  LinkBox,
+  LinkOverlay,
+  HStack,
+  Text,
+  VStack,
+  Flex,
+} from "@chakra-ui/react"
+import { InlineMessage } from "@opengovsg/design-system-react"
+import { AllSitesHeader } from "components/Header/AllSitesHeader"
+import _ from "lodash"
+import { useEffect } from "react"
+import { Link } from "react-router-dom"
+
+import { LOCAL_STORAGE_KEYS } from "constants/localStorage"
+
+import { useLoginContext } from "contexts/LoginContext"
+
+import { useGetAllSites } from "hooks/allSitesHooks"
+
+import elementStyles from "styles/isomer-cms/Elements.module.scss"
+import siteStyles from "styles/isomer-cms/pages/Sites.module.scss"
+
+import { convertUtcToTimeDiff } from "utils/dateUtils"
+
+import { EmptySitesImage, OGPLogoNoText } from "assets"
+import { SiteData } from "types/sites"
+
+const SitesContent = ({ siteNames }: { siteNames?: SiteData[] }) => {
+  const { userId } = useLoginContext()
+  const isEmailLoginUser = !userId
+  const urlLink = isEmailLoginUser ? "dashboard" : "workspace"
+
+  if (siteNames && siteNames.length > 0)
+    return (
+      <>
+        {siteNames.map((siteName) => (
+          <div className={siteStyles.siteContainer} key={siteName.repoName}>
+            <div className={siteStyles.site}>
+              <Link to={`/sites/${siteName.repoName}/${urlLink}`}>
+                <Flex
+                  w="full"
+                  h="70%"
+                  justifyContent="center"
+                  alignItems="center"
+                >
+                  <OGPLogoNoText height="30%" />
+                </Flex>
+                <VStack
+                  w="full"
+                  h="30%"
+                  justifyContent="center"
+                  alignItems="start"
+                  backgroundColor="interaction.mainLight.default"
+                  pl="1rem"
+                  borderRadius="0px 0px 4px 4px"
+                >
+                  <Text fontSize="0.9em">{siteName.repoName}</Text>
+                  <Text fontSize="0.6em" color="base.content.light">
+                    {convertUtcToTimeDiff(siteName.lastUpdated)}
+                  </Text>
+                </VStack>
+              </Link>
+            </div>
+          </div>
+        ))}
+      </>
+    )
+
+  if (siteNames && siteNames.length === 0)
```

Comment:
nit: rather than the pattern
```
if (siteNames && siteNames.length ===0) return showEmptySites`
 if (siteNames && siteNames.length > 0)` return showSites
 return LoadingState
 ```
 
 could we 
```
if (!siteNames) return LoadingState
if (siteNames.length === 0) return return showEmptySites
return showSites
```

Reply:
Yeah that makes sense, https://github.com/isomerpages/isomercms-frontend/pull/1199/commits/540deb4475e306d6ed172b2b6c6fdfae355c8a38!